src/components/NoteDetails/NoteDetailsPage.js

```diff
@@ -1,31 +1,33 @@
 import React from 'react';
 
-import IconButton from '@material-ui/core/IconButton';
-import ArrowBackIcon from '@material-ui/icons/ArrowBack';
-import { Link } from 'react-router-dom';
 import styled from 'styled-components';
+import { colors } from 'colors';
 
 import { NoteDetailsContainer } from 'components/NoteDetails/NoteDetailsContainer';
 
-const Wrapper = styled.div`
-  background-color: #2b2630;
-`;
+import NoteDetailsTopBar from './NoteDetailsTopBar';
+import ModalDeleteComponent from 'components/ModalDelete/ModalDeleteComponent';
+import { useState } from 'react';
 
-const BackButtonWrapper = styled.div`
-  color: red !important;
-  padding: 1rem;
+const Wrapper = styled.div`
+  background-color: ${colors.darkViolet};
 `;
 
 export const NoteDetailsPage = () => {
+  const [open, setOpen] = useState(false);
```

Comment:
It is ambiguous what does `open` mean: since it's in the `NoteDetailsPage` I would assume it means that the NoteDetailsPage is either open or closed.

Let's name it `isDeleteNoteConfirmationDialogOpen`: this way it would be clear what is this state variable used for.